ckanext/xloader/helpers.py

```diff
@@ -25,3 +25,9 @@ def xloader_status_description(status):
         return captions.get(status['status'], status['status'].capitalize())
     else:
         return _('Not Uploaded Yet')
+
+
+def is_xloader_format(resource_format):
+    from ckanext.xloader.plugin import XLoaderFormats
```

Comment:
Can this import be at the top of the module? There's no reason it would clash with other functions.

Reply:
That would cause a circular import, so it has to stay within the method.

Reply:
Perhaps XLoaderFormats shouldn't be within the plugin, then.

Reply:
`XLoaderFormats` moved to utils script to prevent circular imports.